photon-ml/src/integTest/scala/com/linkedin/photon/ml/DriverIntegTest.scala

```diff
@@ -126,21 +135,24 @@ class DriverIntegTest extends SparkTestUtils with TestTemplateWithTmpDir {
     args += CommonTestUtils.fromOptionNameToArg(INTERCEPT_OPTION)
     args += "false"
     appendOffHeapConfig(args, addIntercept = false)
+    args += CommonTestUtils.fromOptionNameToArg(MAX_NUM_ITERATIONS_OPTION)
+    args += MAX_NUM_ITERATIONS.toString
 
     MockDriver.runLocally(args = args.toArray,
-      expectedStages = Array(DriverStage.INIT, DriverStage.PREPROCESSED, DriverStage.TRAINED, DriverStage.VALIDATED, DriverStage.DIAGNOSED),
-      expectedNumFeatures = 13, expectedNumTrainingData = 250, expectedIsSummarized = false)
+      expectedStages = Array(DriverStage.INIT, DriverStage.PREPROCESSED, DriverStage.TRAINED, DriverStage.VALIDATED,
+        DriverStage.DIAGNOSED),
```

Comment:
Same as above, one argument per line if they cannot all fit on one line (alternatively, we could import DriverStage._)